helm/templates/_helpers.tpl

```diff
@@ -27,3 +27,9 @@ Combined Pod labels
 {{- $new := merge .Values.podLabels $combinedLabels }}
 {{- toYaml $new }}
 {{- end -}}
+
+{{- define "engprod.deploymentslice" -}}
```

Comment:
```suggestion
{{- define "elasticsearch-metrics-apiserver.deploymentslice" -}}
```
Making these definitions more consistent with all of the others.

Reply:
@naemono This is really meant to be a generic `engprod` function going forward, so we can add other labels as necessary under our context.  Thoughts on leaving it named that way?

Reply:
> @naemono This is really meant to be a generic `engprod` function going forward, so we can add other labels as necessary under our context. Thoughts on leaving it named that way?

My thoughts for leaving it this way was that this is a public repo and it doesn't mean anything to anyone outside of elastic. but it's not a blocker to me.

Reply:
That's fair. I can shift it.